### PR TITLE
Install new Quicklisp version from 7/7/2022 replacing old one from 2020

### PR DESCRIPTION
**Pull request by westfold**
Fixes problems with latest Ubuntu linux and M1 Macs
Make update-libs.sh more robust
Add dexador https web client library
Add zippy zip file reader library

### CODE REVIEW

File: books/quicklisp/update-libs.sh [hunk 1]
```diff
@@ -49,8 +49,22 @@ fi
 
 if [ -z "$LISP" ]
 then
-    echo "Defaulting LISP to ccl"
-    LISP=ccl
+    if [ ! -z "$ACL2_LISP" ]
```
Comment (bendyarm) [inline books/quicklisp/update-libs.sh:52]: I suggest taking this section out unless there really are users of ACL2_LISP.

File: books/quicklisp/update-libs.sh [hunk 2]
```diff
@@ -49,8 +49,22 @@ fi
...
+    elif [ $(command -v sbcl) ]
+    then
+       echo "Defaulting LISP to sbcl"
+       LISP=sbcl
+    elif [ $(command -v ccl) ]
+    then
+       echo "Defaulting LISP to ccl"
+       LISP=sbcl
+    else
+       echo "Can't find LISP: set $LISP"
```
Comment (bendyarm) [inline books/quicklisp/update-libs.sh:65]: Relayed from ES: $LISP --> \$LISP

File: books/quicklisp/update-libs.sh [hunk 3]
```diff
@@ -49,8 +49,22 @@ fi
...
+       echo "Defaulting LISP to $ACL2_LISP"
+       LISP=$ACL2_LISP
+    elif [ $(command -v sbcl) ]
+    then
+       echo "Defaulting LISP to sbcl"
+       LISP=sbcl
+    elif [ $(command -v ccl) ]
+    then
+       echo "Defaulting LISP to ccl"
+       LISP=sbcl
```
[the remark this answers is not in the document]
Comment (westfold) [reply books/quicklisp/update-libs.sh]: Change made

File: books/quicklisp/update-libs.sh (hunk 2 above)
Comment (westfold) [reply books/quicklisp/update-libs.sh]: Change made

**Comment by MattKaufmann** [comment]
I'm hoping someone will merge this soon (but I don't feel qualified to do so).  It looks like it could be a nice change to get into Version 8.5.

**Pull request merged**